tests/instrumentation-framework/src/main/java/org/hawkular/apm/tests/dockerized/TestScenarioRunner.java

```diff
@@ -123,19 +110,26 @@ public int run(TestScenario testScenario) {
     private void runTestCase(TestScenario testScenario, TestCase testCase,
                              TestEnvironmentExecutor testEnvironmentExecutor) throws TestFailException {
 
-        System.out.println("Executing test case: " + testCase);
+        log.info(String.format("Executing test case: %s", testCase));
 
         String environmentId = null;
+        ApmMockServer apmServer = new ApmMockServer();
+        apmServer.setHost("0.0.0.0");
+        apmServer.setPort(apmServerPort);
+        // disable shut down timer
```

Comment:
Not sure what is the meaning of this comment here. Does it mean that the next line will disable the shutdown timer? It looks like it's the opposite, no?